### PR TITLE
Support translated Options (Server-side)

### PR DESCRIPTION
**Pull request by jlsjonas**
depends on https://github.com/laravel-enso/Select/pull/24

### CODE REVIEW

**Comment by jlsjonas** [comment]
note: documentation will follow once approved

File: src/app/Classes/Attributes/Meta.php [hunk 1]
```diff
@@ -11,6 +11,7 @@ class Meta
         'disabled', 'readonly', 'hidden', 'source', 'format', 'time', 'rows',
         'placeholder', 'trackBy', 'label', 'tooltip', 'symbol', 'precision',
         'thousand', 'decimal', 'positive', 'negative', 'zero', 'resize',
+        'translateFields',
```
Comment (aocneanu) [inline src/app/Classes/Attributes/Meta.php]: let's rename `translateFields` to `translated`.

Add a note to the docs as well. Make sure you refer `options`, not `fields`

**Comment by jlsjonas** [comment]
requested changes made (also in the `Select` repo)

Docs => https://github.com/laravel-enso/Documentation/pull/7

**Comment by aocneanu** [comment]
Thanks for the contribution!

**Pull request merged**